tutorials/polkadot-sdk/testing/spawn-basic-chain.md

```diff
@@ -54,7 +54,7 @@ If successful, you will see the following output:
 
 --8<-- 'code/tutorials/polkadot-sdk/testing/spawn-basic-chain/spawn-network-terminal-01.html'
 
-!!! note 
+!!! tip 
```

Comment:
I understand why you changed this, but I feel like the way the text is worded it sounds more like information, rather than advice.